apps/src/templates/census2017/CensusInaccuracyReview.jsx

```diff
@@ -0,0 +1,400 @@
+import React, { PropTypes, Component } from 'react';
+import {Table} from 'reactabular';
+import Button from '../Button';
+
+export default class CensusInaccuracyReview extends Component {
+  static propTypes = {
+    authenticityToken: PropTypes.string,
+    reportsToReview: PropTypes.array,
+  };
+
+  state = {
+    toReview: undefined,
+    notes: '',
+    resolvedReports: [],
+  };
+
+  formatTeachesCs = (teachesCs) => {
+    return (
+      <div style={{textAlign: 'center', verticalAlign: 'middle'}}>
+        {teachesCs ? teachesCs : "Null"}
+      </div>
+    );
+  };
+
+  formatComment = (comment) => {
+    return (
+      <div>
+        {comment}
+      </div>
+    );
+  };
+
+  mapHistoricSummary = (teaches_cs) => {
+    switch (teaches_cs) {
+      case null:
+        return 'Null';
+      case undefined:
+        return 'No Data';
+      default:
+        return teaches_cs.charAt(0).toUpperCase() + teaches_cs.slice(1).toLowerCase();
+    }
+  };
+
+  formatSchool = (school) => {
+    return (
+      <div>
+        {school.name}
+        <br />
+        ({school.city}, {school.state})
+      </div>
+    );
+  };
+
+  beginReviewButton = (_value, data) => {
+    if (this.state.resolvedReports.includes(data.rowData.id)) {
+      return (
+        <div style={{textAlign: 'center', verticalAlign: 'middle'}}>
+          Review Completed
+        </div>
+      );
+    } else {
+      return (
+        <Button
+          onClick={() => {this.setState({toReview: data.rowData}); $('#report-header')[0].scrollIntoView();}}
+          size="large"
+          text="Review this School"
+        />
+      );
+    }
+  };
+
+  showMainList = () => {
+    this.setState({
+      toReview: undefined,
+      notes: ''
+    });
+    $('#report-header')[0].scrollIntoView();
+  };
+
+  render() {
+    let display;
+    if (this.state.toReview) {
+      display = this.renderReview();
+    } else {
+      display = this.renderTable();
+    }
+
+    return (
+      <div>
+        <h1 id="report-header">
+          Reported Access Report Inaccuracies
+        </h1>
+        {display}
+      </div>
+    );
+  }
+
+  stringForExplanationLabel = {
+    offers_ap: "Offers AP CS?",
+    offers_ib: "Offers IB CS?",
+    consistent_teacher_surveys: "Consistent Teacher/Admin Surveys",
+    state_offering: "Teaches per State Data?",
+    consistent_non_teacher_surveys: "Consistent Non-Teacher/Admin Surveys",
+    inconsistent_surveys: "Inconsistent Surveys",
+    last_years_summary: "Last Year's Summary",
+    two_years_agos_summary: "Two Years Ago's Summary",
+  };
+
+  renderReview = () => {
+    const noData = 'No Data';
+    const school = this.state.toReview.school;
+    const auditData = this.state.toReview.summary_audit;
+    const explanation = auditData.explanation;
+    const highlightedStyle = {
+      backgroundColor: "#8afc9b",
```

Comment:
we have official colors in /util/color if you'd like to make use of that

Reply:
I didn't know about that. I took a look and tried the greens that are defined there but they don't give the desired effect so I'm going to keep this one. Should I be adding it there or just keep it local to this code?

Reply:
If you think it's a color that we'd likely like to use again on the site, then it would be nice to add it to our official color list.

Reply:
It's been pretty ad hoc.  `highlight_green` could family up with the other `_green` colors.

Reply:
Ok. This is done.